src/main/java/mate/academy/spring/controller/MovieController.java

```diff
@@ -0,0 +1,38 @@
+package mate.academy.spring.controller;
+
+import java.util.List;
+import java.util.stream.Collectors;
+import mate.academy.spring.dto.request.MovieRequestDto;
+import mate.academy.spring.dto.response.MovieResponseDto;
+import mate.academy.spring.service.MovieService;
+import mate.academy.spring.service.mapper.MovieMapper;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/movies")
+public class MovieController {
+    private final MovieService movieService;
+    private final MovieMapper movieMapper;
+
+    public MovieController(MovieService movieService, MovieMapper movieMapper) {
+        this.movieService = movieService;
+        this.movieMapper = movieMapper;
+    }
+
+    @PostMapping
+    public MovieResponseDto create(@RequestBody MovieRequestDto movie) {
+        return movieMapper.toDto(movieService.add(movieMapper.toModel(movie)));
+    }
+
+    @GetMapping
+    public List<MovieResponseDto> getAllMovies() {

```

Comment:
```suggestion
    public List<MovieResponseDto> getAllMovies() {
```suggestion
    public List<MovieResponseDto> getAll() {

```
```